spec/fixtures/site/_config.yml

```diff
@@ -0,0 +1,4 @@
+collections:
+  things:
+    output: true
+    permalink: /:path/
```

Comment:
I'm worried that on larger sites, collection support could be a performance concern. What do you think about making it opt-in via the collection config?